### DISCUSSION TITLE
Custom Collection Sorting

### DISCUSSION THREAD

**Discussion by pinovecs** [jellyfin/jellyfin#8759] [General] [answered]
Is there a method to create a custom sort of a Movie collection without using the "sort title" metadata tag or if that tag is used it doesn't impact other collections.

Example: I have a custom collection of all Marvel movies.  Not just the MCU movies but the Fox X-Men movies the Sony Spider-Man movie and others from various studios over time.  I want the MCU movies first so right now Iron Man (2008) has a "sort title" metadata of 1 and they sort from there.  Then after all the MCU I have the first X-Men movie...etc, etc.

The issue is these movies can't be included in any other general collections (Action, Sci-Fi, Fantasy, etc) without being at the very top of the list.

Is there a way to manipulate a collection as I'm trying to do?

Thansks!

**Comment by Darkhost1999** [comment, marked as answer]
Per collection configuration is an interesting idea. Yet I don't believe this is possible at this exact time.

**Reply by JPVenson** [reply]
This is correct. Collections can be sorted individually based on the existing tags but there is no custom per-collection sorting tag. If that is a desirably feature, please formulate a features.jellyfin.org request
